core-web/libs/block-editor/src/lib/extensions/asset-form/asset-form.extension.ts

```diff
@@ -19,6 +19,11 @@ declare module '@tiptap/core' {
             openAssetForm: (data: { type: EditorAssetTypes }) => ReturnType;
             closeAssetForm: () => ReturnType;
             insertAsset: (data: { type: EditorAssetTypes; payload }) => ReturnType;
+            insertAssetAt: (data: {
```

Comment:
Can we unify this two? and just set a default position when not passed?

Reply:
Sure!